### PR TITLE
Autoclean fixes and docs

### PR DESCRIPTION
**Pull request by rustyrussell**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -30,6 +30,7 @@ Core Lightning Documentation
    :caption: Manpages
 
    lightning-addgossip <lightning-addgossip.7.md>
+   lightning-autoclean-once <lightning-autoclean-once.7.md>
```
Comment (cdecker) [inline doc/index.rst:33]: Looks like this file is missing.

**Review by cdecker — CHANGES_REQUESTED** [review]
Missing a doc markdown file, but otherwise gtg :-)

File: doc/index.rst (hunk 1 above)
Comment (rustyrussell) [reply doc/index.rst:33]: Doh!!!

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/5775/commits/492d99bb26347e283cd2443cbdb0cc6ec0a07e96

**Pull request merged**